ArcGISRuntimeSDKQt_QMLSamples/Analysis/StatisticalQueryGroupSort/OptionsPage.qml

```diff
@@ -207,7 +207,6 @@ Rectangle {
                             highlightMoveVelocity: 1000000
 
                             delegate: Item {
-                                width: parent.width
```

Comment:
Found one more null width error: `qrc:/Samples/Analysis/StatisticalQueryGroupSort/OptionsPage.qml:279:33: TypeError: Cannot read property 'width' of null`

(Found when scrolling to the bottom of the "Group Field(s)" list)

Reply:
Once that's resolved it's good to merge